src/Shared/runtime/Http3/QPack/QPackDecoder.cs

```diff
@@ -44,7 +42,7 @@ private enum State
         //+---+---------------------------+
         //|      Compressed Headers     ...
         private const int RequiredInsertCountPrefix = 8;
-        private const int BaseMask = 0x80;
+        //private const int BaseMask = 0x80;
```

Comment:
Nit:

```suggestion
```

Was this doing anything before? I see it was used to set `prefixInt`, but it doesn't appear it was used in the `State.Base` case. I guess that's probably why you removed it.